wls-wahlvorbereitung-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/service/urnenwahlvorbereitung/UrnenwahlvorbereitungService.java

```diff
@@ -35,4 +38,22 @@ public Optional<UrnenwahlvorbereitungModel> getUrnenwahlvorbereitung(@P("wahlbez
 
         return dataFromRepo.map(urnenwahlvorbereitungModelMapper::toModel);
     }
+
+    @PreAuthorize(
+        "hasAuthority('Wahlvorbereitung_BUSINESSACTION_PostUrnenwahlVorbereitung')"
+                + "and @bezirkIdPermisionEvaluator.tokenUserBezirkIdMatches(#vorbereitungToSet.wahlbezirkID, authentication)"
+    )
+    public void setUrnenwahlvorbereitung(@P("vorbereitungToSet") final UrnenwahlvorbereitungModel vorbereitungToSet) {
+        log.debug("#postUrnenwahlVorbereitung");
```

Comment:
```suggestion
        log.debug("#setUrnenwahlvorbereitung");
```

would be more appropriate for this method, but maybe you wanted to have the name of the Controller-Method here?

Reply:
i just copied the old code. I dont know if it is used in existing queries, so i deciced to keep the message.

Reply:
keeped for compatibility